src/main/java/se/bjurr/violations/lib/example/CopyOfMyClass.java

```diff
@@ -0,0 +1,19 @@
+package se.bjurr.violations.lib.example;
+
+public class CopyOfMyClass {
+ public static String CONstant = "yes";
+
+ void npe(String a, String b) {
+  if (a == null) {
+   System.out.println();
+  } else {
+
+  }
+  a.length();
+ }
+
+ @Override
+ public boolean equals(Object obj) {
```

Comment:
Reporter: PMD
Rule: OverrideBothEqualsAndHashcode
Severity: WARN
File: /home/travis/build/tomasbjerre/violations-test/src/main/java/se/bjurr/violations/lib/example/CopyOfMyClass.java L16
 

Basic http://pmd.sourceforge.net/pmd-5.2.3/pmd-java/rules/java/basic.html#OverrideBothEqualsAndHashcode

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>